dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,32 @@
+public class Demo {
```

Comment:
Hello from techdebt analyzer
<!-- lookout footnote separator -->
_Comment made by the analyzer techdebt._